### DISCUSSION TITLE
我联系你们了，请问关于“毛泽东博览”这个网站的爬取问题？

### DISCUSSION THREAD

**Discussion by CiangCing14** [banned-historical-archives/banned-historical-archives.github.io#3944] [General]
[code.zip](https://github.com/banned-historical-archives/banned-historical-archives.github.io/files/14353972/code.zip)

**Comment by CiangCing14** [comment]
目录树文件下载完后，下载文件：
[code-2.zip](https://github.com/banned-historical-archives/banned-historical-archives.github.io/files/14354125/code-2.zip)

**Reply by CiangCing14** [reply]
下载不稳定。

**Comment by CiangCing14** [comment]
目录树文件下载完后，下载文件（自动重试）：
[code-3.zip](https://github.com/banned-historical-archives/banned-historical-archives.github.io/files/14354216/code-3.zip)

**Comment by CiangCing14** [comment]
已下载完毕的索引文件：
[index.zip](https://github.com/banned-historical-archives/banned-historical-archives.github.io/files/14354197/index.zip)

**Comment by CiangCing14** [comment]
第一批爬取结果还需要完善。

**Comment by CiangCing14** [comment]
[code-4.zip](https://github.com/banned-historical-archives/banned-historical-archives.github.io/files/14354627/code-4.zip)

**Comment by CiangCing14** [comment]
https://github.com/CiangCing14/mbl

**Reply by CiangCing14** [reply]
如果有消息了请回复，不要只是一个点赞，我联系piratebook1了，或者，提供联系方式也是可以的。

**Reply by banned-historical-archives** [reply]
抱歉暂没时间处理这个事情，另外据说mzdbl已经可以正常运行

**Reply by CiangCing14** [reply]
这个网站已经不是第一次关闭了。

**Comment by CiangCing14** [comment]
文件更多内容的下载代码，但是因为速度问题，很慢：
[code-4.zip](https://github.com/banned-historical-archives/banned-historical-archives.github.io/files/14366735/code-4.zip)

**Comment by CiangCing14** [comment]
> 抱歉暂没时间处理这个事情，另外据说mzdbl已经可以正常运行

这个网站已经不是第一次关闭了。

**Comment by banned-historical-archives** [comment]
https://huggingface.co/datasets/banned-historical-archives/banned-historical-archives/tree/main/todo/%E6%9C%AA%E5%88%86%E7%B1%BB22

**Reply by CiangCing14** [reply]
这真是感激不尽。
